routers/web/repo/issue.go

```diff
@@ -246,6 +246,12 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 		isShowClosed = true
 	}
 
+	archived := ctx.FormBool("archived")
+	showArchivedLabels := "false"
+	if archived {
+		showArchivedLabels = "true"
+	}
+
```

Comment:
Could you check what will be the value of `ctx.FormBool("archived")` if you don't pass into query params,

I guess you don't need `showArchivedLabels` varable extra to save the state . You can use `archived` variable. directly

Reply:
It is "false", type is bool.
improved in https://github.com/go-gitea/gitea/pull/27302/commits/8508daf6dcdf785bc2c864c2ad944639d52512a8

Reply:
you can use `strings` package . `strings.FormatBool`